src/group_theory/group_action/basic.lean

```diff
@@ -194,6 +227,48 @@ rfl
   mul_action.stabilizer G ((1 : G) : quotient H) = H :=
 by { ext, simp [quotient_group.eq] }
 
+local notation `Ω` := (quotient $ orbit_rel α β)
+
+/-- **Burnside's lemma** : given a group `G` acting on a set `X`, there is a bijection between the
+disjoint union of all `{x ∈ X | g • x = x}` for `g ∈ G` and the product `G × X/G`, where `X/G`
+denotes the quotient of `X` by the relation `orbit_rel G X`. -/
```

Comment:
This docstring sounds like you're proving a lemma while you are constructing data.